test/unittests/state_rlp_test.cpp

```diff
@@ -377,3 +377,74 @@ TEST(state_rlp, tx_to_rlp_eip1559_invalid_v_value)
 
     EXPECT_THROW(rlp::encode(tx), std::invalid_argument);
 }
+
+TEST(state_rlp, tx_to_rlp_eip2930_invalid_v_value)
+{
+    state::Transaction tx{};
+    tx.kind = evmone::state::Transaction::Kind::eip2930;
+    tx.data = ""_hex;
+    tx.gas_limit = 1;
+    tx.max_gas_price = 1;
+    tx.max_priority_gas_price = 1;
+    tx.sender = 0x0000000000000000000000000000000000000000_address;
+    tx.to = 0x0000000000000000000000000000000000000000_address;
+    tx.value = 0;
+    tx.access_list = {};
+    tx.nonce = 47;
+    tx.r = 0x0000000000000000000000000000000000000000000000000000000000000000_u256;
+    tx.s = 0x0000000000000000000000000000000000000000000000000000000000000000_u256;
+    tx.v = 2;
+    tx.chain_id = 1;
+
+    EXPECT_THROW(rlp::encode(tx), std::invalid_argument);
```

Comment:
Can you try these matchers? https://github.com/google/googletest/pull/2904

Reply:
It requires `GTest => 1.11.0`. Last available version of GTest on `hunter` is 1.7.0.

Reply:
Nice find.